account_financial_report_webkit/report/common_partner_reports.py

```diff
@@ -89,17 +89,20 @@ def _get_first_special_period(self):
     def _get_period_range_from_start_period(self, start_period,
                                             include_opening=False,
                                             fiscalyear=False,
-                                            stop_at_previous_opening=False):
+                                            stop_at_previous_opening=False,
+                                            **kwargs):
```

Comment:
Is there a specific reason you used the `kwargs` key instead of a named param, or what is the benefit of being able to pass more parameters to this method?

Reply:
yes, this is override of [this method](https://github.com/EssentNovaTeam/account-financial-reporting/pull/3/files#diff-07dff3641d9dda2dfeb17d89bcd41ba4L314). I needed to find a way to optionally exclude `first special period` without extending base method with the specific named parameter which  is needed here as it does not make sense to do something like that, so the only other option was to go with ``kwargs``.

Reply:
Is this something that can be added to the context instead?

Reply:
That was my first approach but unfortunately these classes don't have context attribute.

Reply:
It's painful, I know, but from my perspective this was the only way to go in order to keep changes on minimum.

Reply:
Further more, context/localcontext comes with class `report_sxw.rml_parse` and that class is extended by the actual report classes (i.e. `PartnerBalance`, `PartnersLedger`, etc) but the classes I extended here do not have such an attribute.

Reply:
Thanks for the clarification!